src/change_stream.ts

```diff
@@ -304,83 +326,28 @@ export class ChangeStream extends EventEmitter {
       const cursor = this.cursor;
 
       return cursor.close(err => {
-        ['data', 'close', 'end', 'error'].forEach(event => cursor.removeAllListeners(event));
+        ['data', 'close', 'end', 'error'].forEach(event =>
```

Comment:
nit: I would invert this, if the cursor stream is null then you don't want to needlessly loop through the event list:
```
if (this[kCursorStream]) {
   ['data', 'close', 'end', 'error'].forEach(event => this[kCursorStream]?.removeAllListeners(event));
}
```

Reply:
also, you use defined types for this below (`[CursorStream.DATA, CursorStream.CLOSE, CursorStream.END, CursorStream.ERROR]`)